csharp/extractor/Semmle.Extraction.Tests/BUILD.bazel

```diff
@@ -4,7 +4,8 @@ load(
 )
 
 codeql_xunit_test(
-    name = "Semmle.Extraction.Tests",
+    # short name as we run into long path limitations on Windows
+    name = "t",
```

Comment:
Maybe use different short names for each project?

Reply:
Sure! Let's see if 4 characters are too much already or not.